src/argilla_sdk/_models/_settings/_fields.py

```diff
@@ -48,13 +49,6 @@ def __title_default(cls, title: str, info: ValidationInfo) -> str:
         log(f"TextField title is {validated_title}")
         return validated_title
 
-    @field_serializer("id", when_used="unless-none")
+    @field_serializer("id", "dataset_id", when_used="unless-none")
```

Comment:
perhaps it is cleaner to add the `field_serializer` for `id` within the base`ResourceModel` class and only add `dataset_id` here? Might introduce some duplicate code though.